sale_blanket_order/models/blanket_orders.py

```diff
@@ -0,0 +1,649 @@
+                {
+                    "amount_untaxed": currency.round(amount_untaxed),
+                    "amount_tax": currency.round(amount_tax),
+                    "amount_total": amount_untaxed + amount_tax,
+                }
+            )
+
+    name = fields.Char(default="Draft", readonly=True, copy=False)
+    partner_id = fields.Many2one(
+        "res.partner",
+        string="Partner",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    line_ids = fields.One2many(
+        "sale.blanket.order.line", "order_id", string="Order lines", copy=True
+    )
+    line_count = fields.Integer(
+        string="Sale Blanket Order Line count",
+        compute="_compute_line_count",
+        readonly=True,
+    )
+    product_id = fields.Many2one(
+        "product.product", related="line_ids.product_id", string="Product",
+    )
+    pricelist_id = fields.Many2one(
+        "product.pricelist",
+        string="Pricelist",
+        required=True,
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    currency_id = fields.Many2one("res.currency", related="pricelist_id.currency_id")
+    payment_term_id = fields.Many2one(
+        "account.payment.term",
+        string="Payment Terms",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    confirmed = fields.Boolean(copy=False)
+    state = fields.Selection(
+        selection=[
+            ("draft", "Draft"),
+            ("open", "Open"),
+            ("done", "Done"),
+            ("expired", "Expired"),
+        ],
+        compute="_compute_state",
+        store=True,
+        copy=False,
+    )
+    validity_date = fields.Date(readonly=True, states={"draft": [("readonly", False)]})
+    client_order_ref = fields.Char(
+        string="Customer Reference",
+        copy=False,
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    note = fields.Text(readonly=True, states={"draft": [("readonly", False)]})
+    user_id = fields.Many2one(
+        "res.users",
+        string="Salesperson",
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    team_id = fields.Many2one(
+        "crm.team",
+        string="Sales Team",
+        change_default=True,
+        default=_get_default_team,
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    company_id = fields.Many2one(
+        "res.company",
+        string="Company",
+        default=_default_company,
+        readonly=True,
+        states={"draft": [("readonly", False)]},
+    )
+    sale_count = fields.Integer(compute="_compute_sale_count")
+
+    fiscal_position_id = fields.Many2one(
+        "account.fiscal.position", string="Fiscal Position"
+    )
+
+    amount_untaxed = fields.Monetary(
+        string="Untaxed Amount",
+        store=True,
+        readonly=True,
+        compute="_compute_amount_all",
+        track_visibility="always",
```

Comment:
```suggestion
           tracking=True,
```
track_visibility is not available anymore